src/V3AstNodes.cpp

```diff
@@ -1639,6 +1639,18 @@ void AstInitArray::cloneRelink() {
         if (it->second->clonep()) it->second = it->second->clonep();
     }
 }
+
+bool AstInitArray::same(const AstNode* samep) const {
+    const AstInitArray* sp = VN_DBG_AS(samep, InitArray);
```

Comment:
We probably should cache this result in a VIsCached in the AstInitArray.

Reply:
Since the function is used to compare random nodes probably it's not convenient to cache the results?